StudyHub/StudyHub.BLL/Services/PaginationService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using AutoMapper;
+using StudyHub.BLL.Services.Interfaces;
+using StudyHub.Common.DTO.User;
+using StudyHub.Entities;
+
+namespace StudyHub.BLL.Services;
+
+public class PaginationService : IPaginationService
+{
+    private readonly IMapper _mapper;
+
+    public PaginationService(IMapper mapper)
+    {
+        _mapper = mapper;
+    }
+
+    public PageList Pagination(List<User> users, int page, int pageSize)
+    {
+        var pageList = new PageList();
+        var pageUsers = users.Skip((page - 1) * pageSize).Take(pageSize);
```

Comment:
first, ,make pageUsers and then pageList like:
pageList = new PageList
{
TotalCount = ...
}

Reply:
wdym?

Reply:
this need to not write pageList.SomeProps on creating